lib/DDG/Spice/Cryptocurrency.pm

```diff
@@ -150,10 +140,14 @@ sub checkCurrencyCode {
         if (exists($excludedCurrencies{$from})) {
             return;
         }
-        $endpoint = 'secondaries';
-        # API does not return the display name of the currency, so we must include it
-        $query = $normalized_number . $currDisplayName;
-        $query2 = $from;
+
+        if($to eq "") {
+            $to = "BTC";
+        }
```

Comment:
Not critical to launch with, but what do you think about defaulting to local currency instead like the other IA?